docs/registry.md

```diff
@@ -34,11 +34,12 @@ Proposals can be of different type and include (`proposal_metadata`) different d
 2. Proposal to update the set of successful-proposal receiver contract addresses.
 This proposal takes a parameter that has two constructors. The proposal can add to,
 or remove from the set of addresses.
-3. Transfer proposals, that can transfer XTZ or tokens as well as update the
+3. Transfer proposals, that can transfer XTZ or tokens (FA2 as well as FA1.2) as well as update the
 registry. This includes:
    - a list of `update` items parameterized by `k` and `v` types. Each `update` item contains a key of the type `k` and a new value of the type `option v`.
-   - a list of items where each item contains:
+   - for FA2 transfers a list of items where each item contains:
    `or %transfers (pair (mutez %amount) (address %recipient)) (pair (address %fa2) (list (pair (address %from_) (list %txs (pair (address %to_) (pair (nat %token_id) (nat %amount)))))))` specifies what transfer to make. The left part is used for XTZ transfers, the right part is used for FA2 transfers.
```

Comment:
(continuing from the previous comment)

this line seems to be outdated too and IDK why FA1.2 wouldn't be considered as an option here as well